pkg/xen-tools/initrd/init-initrd

```diff
@@ -13,9 +13,21 @@ then
     exit 1
 fi
 
+MOUNT_TAGS=`cat /proc/cmdline | grep -o '\bvolumes=[^ ]*' | cut -d = -f 2`
+IFS=';' read -r -a mounts <<< "$MOUNT_TAGS"
```

Comment:
This is not valid posix sh, it only works in bash, which we cannot rely on having here. Can you convert this to straight sh?

Reply:
Also, it looks like you separated the `volumes` using `,` in `hypervisor/xen.go`, but are splitting on `;` here?